### PR TITLE
Move pagination translations to active_admin scope

### LINKED ISSUE #8215: v4 beta 3: Kaminari translation override may affect the host application
Not sure that this should be part of activeadmin's `locale.yml` as it is

Kaminari may be used also in the "host" application, so this will override translations of the host front-end

https://github.com/activeadmin/activeadmin/blob/4318a7a5ca1845e08437e8c0d0321ea659b6aa08/config/locales/en.yml#L2-L5

Introduced in 9999e80cadaf16a8d1d0a6e092229107979f5760

### PR DESCRIPTION
**Pull request by tagliala**
Active Admin uses the Kaminari gem for pagination, which may also be used by the host application. Moving the translations from `views.pagination` to `active_admin.pagination` prevents potential conflicts.

Fix #8215

### CODE REVIEW

File: app/views/active_admin/kaminari/_next_page.html.erb
Comment (tagliala) [inline app/views/active_admin/kaminari/_next_page.html.erb:11]: not sure about this, since the default template is using this translation in a screen reader only span, wouldn't be better to remove `html_safe`?

Comment (javierjulio) [reply app/views/active_admin/kaminari/_next_page.html.erb:11]: Yes, I think so. I left this as is because it's what Kaminari used but then it had to as it used HTML entities for arrows which we don't want. We just need the plain text. I'll leave it up to you to make that change if you want. This is all set to merge either way so thank you!

Comment (javierjulio) [reply app/views/active_admin/kaminari/_next_page.html.erb:11]: I just rebased so the tests should now pass. Sorry about that. I was adding the changelog from 3.2.0 into default branch and forgot we run tests on the changelog but no more.

**Review by javierjulio — APPROVED** [review]
Thanks! Feel free to remove the `.html_safe` if you want. I can release beta4 once you merge.

**Review by javierjulio — APPROVED** [review]
Thanks! ❤️

**Pull request merged**
